### PR TITLE
Add basic setup and PoC

### PR DESCRIPTION
**Pull request by gregorw**
This is a basic setup for SDX. It is based on libSass and has no Ruby dependency, uses sass linting and deploys to Github Pages: http://swisscom.github.io/sdx/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sdx",
+  "version": "0.0.0",
+  "description": "Swisscom Digital Experience",
+  "main": "index.js",
+  "scripts": {
```
Comment (mastertinner) [inline package.json]: a script calling itself?^^

File: README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
 Swisscom Digital Experience
+
+
+## Installation
+
+Install Ruby Sass by running
+
+`bundle install`.
+
+This requires that you install Gem and Bundler before.
```
Comment (mastertinner) [inline README.md]: Here we should provide links to the respective installation instructions: e.g. "\[Bundler\](https://bundler.io/)"

File: Gulpfile.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 var gulp = require('gulp');
 var sass = require('gulp-ruby-sass');
+var jade = require('gulp-jade');
```
Comment (mastertinner) [inline Gulpfile.js]: jade has been renamed to "pug". we should use [gulp-pug](https://www.npmjs.com/package/gulp-pug)

File: README.md (hunk 2 above)
Comment (gregorw) [inline README.md]: @mastertinner Danke für deine guten Inputs. Ich arbeite morgen daran weiter.

File: .ruby_version [hunk 4]
```diff
@@ -0,0 +1 @@
+2.3.0
```
Comment (mastertinner) [inline .ruby_version]: please update to Ruby 2.3.1

File: Gulpfile.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
+var gulp = require('gulp');
```
Comment (mastertinner) [inline Gulpfile.js]: Can we use the ES2015 `const` here?

File: Gulpfile.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    style: 'expanded',
+    loadPath: [bootstrap.src + 'assets/stylesheets']
+  }
+};
+
+var html = {
+  src: sdx.src + '**/*.jade'
+}
+
+gulp.task('fonts', function() {
```
Comment (mastertinner) [inline Gulpfile.js]: Can we use the ES2015 arrow function expressions here?

**Comment by mastertinner** [comment]
Regarding the icon fonts: I think there's an official Swisscom icon font with our pictograms, no? Can we replace the Glyphicons included in Bootstrap with the Swisscom pictograms?

File: .ruby_version (hunk 4 above)
Comment (gregorw) [inline .ruby_version]: I’d like to drop the Ruby dependency all together and use LibSass instead.

File: Gulpfile.js (hunk 5 above)
Comment (gregorw) [inline Gulpfile.js]: Sounds good 👍

**Comment by gregorw** [comment]
@mastertinner Thanks for all the feedback. I have the feeling that it’s not ideal that I keep adding stuff to this PR—it would be much better to keep it small and merge it fast. The thing is that I have to hand something in soon. So, my suggestion is to finish the PoC (Buttons, Checkboxes, Github-Page) within this PR and then do the review before merging into master.

**Comment by mastertinner** [comment]
Sounds good. Thanks, @gregorw!

**Comment by gregorw** [comment]
@mastertinner Ready for review if you like :)

**Pull request merged**